java/DoublyLinkedList.java

```diff
@@ -0,0 +1,123 @@
+public class DoublyLinkedList {
+
+  public Node head;
+  public Node tail;
+  public int length = 0;
+
+  private class Node {
+    private Node next;
+    private Node prev;
+    private String data;
+
+    // Constructor
+    public Node(Node initNext, Node initPrev, String initData){
+      this.next = initNext;
+      this.prev = initPrev;
+      this.data = initData;
+    }
+
+    // Getters
+    public Node getNext() {
+      return this.next;
+    }
+    public Node getPrev() {
+      return this.prev;
+    }
+    public String getData() {
+      return this.data;
+    }
+
+    // Setters
+    public void setNext(Node setTo) {
+      this.next = setTo;
+    }
+    public void setPrev(Node setTo) {
+      this.prev = setTo;
+    }
+    public void setData(String setTo) {
+      this.data = setTo;
+    }
+  } // End of Node class
+
+  public String insertNode(String data){
+    Node expo = new Node (null, null, data);
```

Comment:
Java allows you to implement multiple constructors (to avoid cases where you have to pass in `null` arguments to a constructor. 

[Example](http://stackoverflow.com/questions/581873/best-way-to-handle-multiple-constructors-in-java).

Try adding another constructor that takes in only `data`.